constant 2435016766 overflows int32

Hello, I am getting the following error using btrfs on Raspberry Pi 4

```
# github.com/dennwc/btrfs
../go/pkg/mod/github.com/dennwc/btrfs@v0.0.0-20190517175702-d917b30ff035/utils.go:18:19: constant 2435016766 overflows int32
make: *** [Makefile:8: get] Error 2
```
Is there a corresponding solution? ?

--- a/utils.go
+++ b/utils.go
@@ -15,7 +15,7 @@ func isBtrfs(path string) (bool, error) {
 	if err := syscall.Statfs(path, &stfs); err != nil {
 		return false, &os.PathError{Op: "statfs", Path: path, Err: err}
 	}
-	return stfs.Type == SuperMagic, nil
+	return int64(stfs.Type) == SuperMagic, nil
 }
 
 func findMountRoot(path string) (string, error) {

